hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ProcessorTasklet.java

```diff
@@ -197,10 +198,11 @@ private OutboxImpl createOutbox(@Nonnull OutboundCollector ssCollector) {
 
     @Override
     public void init() {
-        if (serializationService.getManagedContext() != null) {
+        ManagedContext managedContext = serializationService.getManagedContext();
+        if (managedContext != null) {
             Processor toInit = processor instanceof ProcessorWrapper
                     ? ((ProcessorWrapper) processor).getWrapped() : processor;
-            Object initialized = serializationService.getManagedContext().initialize(toInit);
+            Object initialized = managedContext.initialize(toInit);
```

Comment:
shouldn't use the Util method here too?

Reply:
I guess we want to have object equality here, because we may be using the processor elsewhere ?